app/models/compute_resources/foreman/model/vmware.rb

```diff
@@ -1,5 +1,6 @@
 require 'fog_extensions/vsphere/mini_servers'
 require 'foreman/exception'
+require 'rbvmomi'
```

Comment:
I don't think this require is needed, as long as fog-vsphere has been required this should work. Notice how we use RbVmomi in this class w/o requiring it, line 262 for instance.